core/src/main/java/com/sequenceiq/cloudbreak/service/stack/flow/StackOperationService.java

```diff
@@ -282,22 +285,12 @@ public FlowIdentifier updateNodeCountStartInstances(Stack stack, InstanceGroupAd
                 updateNodeCountValidator.validateServiceRoles(stackWithLists, instanceGroupAdjustmentJson);
                 updateNodeCountValidator.validateStackStatus(stackWithLists);
```

Comment:
This would likely need to be changed eventually? i.e. the stack/cluster would be in a NODE_FAILURE state (except for the temporary code above which forces it into AVAILABLE state)